test/unit/edition_test.rb

```diff
@@ -21,6 +21,11 @@ class EditionTest < ActiveSupport::TestCase
     assert_kind_of Document, edition.document
   end
 
+  test "adds auth bypass id to a newly created edition" do
+    edition = build(:edition)
+    assert_not_nil edition.set_auth_bypass_id
```

Comment:
```suggestion
    assert_not_nil set_auth_bypass_id
```
It looks like this is currently testing the method only, rather than ensuring the `before_create` is triggered correctly and the value saved into the database.